dataSource1/src/main/java/QuizGateway.kt

```diff
@@ -0,0 +1,4 @@
+interface QuizGateway {
+    fun getSessionToken(): String
+    fun getQuestions(amount: Int = 10, category: String, difficulty: String, type: String): String
+}
```

Comment:
`category`, `difficulty`, `type` should be enums

Reply:
How? Do I need to make two models or just wait until you merge other PR so I can use EnumClasses